frontend/server/src/DAO/Runs.php

```diff
@@ -244,6 +246,66 @@ final public static function getAllRuns(
                 $where[] = 's.verdict = ?';
                 $val[] = $verdict;
             }
+        } else {
+            if (!is_null($execution)) {
+                if ($execution === 'EXECUTION_INTERRUPTED') {
+                    $where[] = 's.verdict IN(?, ?, ?, ?, ?, ?)';
+                    $val[] = 'ML';
+                    $val[] = 'MLE';
+                    $val[] = 'TLE';
+                    $val[] = 'OLE';
+                    $val[] = 'TO';
+                    $val[] = 'OL';
+                } elseif ($execution === 'EXECUTION_RUNTIME_ERROR') {
+                    $where[] = 's.verdict IN(?, ?)';
+                    $val[] = 'RE';
+                    $val[] = 'RTE';
+                } elseif ($execution === 'EXECUTION_RUNTIME_FUNCTION_ERROR') {
+                    $where[] = 's.verdict IN(?, ?)';
+                    $val[] = 'OF';
+                    $val[] = 'RFE';
+                } elseif ($execution === 'EXECUTION_COMPILATION_ERROR') {
+                    $where[] = 's.verdict = ?';
+                    $val[] = 'CE';
+                } elseif ($execution === 'EXECUTION_VALIDATOR_ERROR') {
+                    $where[] = 's.verdict = ?';
+                    $val[] = 'VE';
+                } elseif ($execution === 'EXECUTION_JUDGE_ERROR') {
+                    $where[] = 's.verdict = ?';
+                    $val[] = 'JE';
+                } else {
+                    $where[] = 's.verdict IN(?, ?, ?)';
+                    $val[] = 'AC';
+                    $val[] = 'WA';
+                    $val[] = 'PA';
+                }
```

Comment:
Como ya estarías utilizando un array asociativo, podrías simplificar el código de la siguiente manera: 
```suggestion
                $executionArgs = \OmegaUp\Controllers\Run::EXECUTION[$execution];
                $placeholders = array_fill(0, count($executionArgs), '?');
                $placeholders = join(',', $placeholders);

                $where[] = "s.verdict IN ({$placeholders})";
                $val = array_merge($val, $executionArgs);
```
Donde la variable `$placeholders` rellenará una cadena de texto con la cantidad de datos variables que se mandarán en la clausula `where`. 

Y en los parámetros de la variable `$val` se asignarán los valores del array que se obtenga de `\OmegaUp\Controllers\Run::EXECUTION[$execution]`